test/utils/TestUtil.scala

```diff
@@ -54,13 +54,20 @@ trait TestUtil extends UnitSpec with GuiceOneAppPerSuite with MaterializerSuppor
   val testEmail = "test@email.co.uk"
   val testWebsite = "https://www.test-website.co.uk"
 
+  val testLandline = "012345678910"
```

Comment:
Some example phone numbers already exist in `BaseTestConstants`. Best to reuse those I think